### ISSUE TITLE
SELECT DISTINCT ON COLUMN

### ISSUE THREAD

**Issue by DevX86** [go-jet/jet#113]
Attempting to-do a

```
SELECT 
    COUNT(DISTINCT column_name),
    another_column,
    another_column
...
```

Is this currently possible @go-jet ? I saw SELECT(table.Table.ColumnName)).DISTINCT() But that wouldn't work in this scenario.

**Comment by go-jet** [comment]
Currently, this is only possible with [raw expressions](https://github.com/go-jet/jet/wiki/Expressions#raw-expression):
```go
s := SELECT(
	Raw("COUNT(DISTINCT(table.column))")).AS("alias")
FROM(table)
```

**Comment by DevX86** [comment]
Seems to work, that's a pretty cool way to implement things until we have it typed as we don't have to make the entire statement raw, just the part that's unique.

**Comment by go-jet** [comment]
Aggregate `DISTINCT` operator included in Release 2.7.0. 
Sample usage:
https://github.com/go-jet/jet/blob/3e802f8955f2f8e85852661f1c7d89ef72503d38/tests/postgres/select_test.go#L1195-L1203
